test/it/ShutMucrSpec.scala

```diff
@@ -23,7 +24,7 @@ class ShutMucrSpec extends IntegrationSpec {
 
   "Enter MUCR Page" when {
     "GET" should {
-      "return 200" in {
+      "throw feature disabled" in {
```

Comment:
Nit: now it throws Invalid Feature State ATM. But this is not that important here.